src/logic/resizeRatioCalculator.ts

```diff
@@ -0,0 +1,16 @@
+import { ResizeRatio, IResizeRatioCalculator } from '../types/asset-analyzer'
```

Comment:
I was going to suggest going the other way around... renaming `resizeRatioCalculator.ts` to `resize-ratio-calculator.ts`, rather than the renamed file above (`assetConverter.ts`).

Reply:
I'll updating these names and renaming everything to kebab in a further PR if we agree so, thank you.